### DISCUSSION TITLE
Grouping and count by createdAt month

### DISCUSSION THREAD

**Discussion by rahmat-dev** [prisma/prisma#19272] [Q&A]
I need data on the number of users who register every month to be displayed on the chart of my dashboard page. How do I group users based on the month of the createdAt field and calculate it with prisma?

Note: I use MongoDB

**Comment by nurul3101** [comment]
Hello @rahmat-dev 👋 

Can you share your Prisma Schema file or your model on which you want to calculate number of users?

**Reply by nurul3101** [reply]
@rahmat-dev I am checking to see if you had a chance to look at my last response. Please let us know in case you need any assistance.

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have the a similar problem :pray:
